GrievanceManagement/src/test/java/com/gms/dto/TicketSaveInDTOTest.java

```diff
@@ -27,21 +27,21 @@ public void testGetterAndSetter() {
         ticketSaveInDTO.setDescription("keyboard is not working");
         assertEquals("keyboard is not working", ticketSaveInDTO.getDescription());
         assertNull(ticketSaveInDTO.getTicketType());
-        ticketSaveInDTO.setTicketType(TicketType.Feedback);
-        assertEquals(TicketType.Feedback, ticketSaveInDTO.getTicketType());
-        assertEquals(0,ticketSaveInDTO.getDepartmentId());
-        ticketSaveInDTO.setDepartmentId(1);
+        ticketSaveInDTO.setTicketType(TicketType.FEEDBACK);
+        assertEquals(TicketType.FEEDBACK, ticketSaveInDTO.getTicketType());
+        assertNull(ticketSaveInDTO.getDepartmentId());
+        ticketSaveInDTO.setDepartmentId(1l);
         assertEquals(1, ticketSaveInDTO.getDepartmentId());        
-        assertEquals(0,ticketSaveInDTO.getUserId());
-        ticketSaveInDTO.setUserId(1);
+        assertNull(ticketSaveInDTO.getUserId());
+        ticketSaveInDTO.setUserId(1l);
         assertEquals(1, ticketSaveInDTO.getUserId());    
     }
+    
     @Test
-    @DisplayName("TicketSaveInDTOTest toString, equals&hashCode")
     public void testEqualAndHashCodeAndToString() {
```

Comment:
how this test case work

Reply:
how can we test tostring and equalhash code method in same

Reply:
yes sir i have tested both method in a seme method